pkg/uploader/provider/kopia.go

```diff
@@ -189,7 +190,7 @@ func (kp *kopiaProvider) RunBackup(
 	)
 
 	log.Debugf("Kopia backup finished, snapshot ID %s, backup size %d", snapshotInfo.ID, snapshotInfo.Size)
-	return snapshotInfo.ID, false, snapshotInfo.Size, nil
+	return snapshotInfo.ID, false, snapshotInfo.Size, snapshotInfo.IncrementalSize, nil
```

Comment:
See https://github.com/vmware-tanzu/velero/pull/9357/files#r2450279613 and https://github.com/vmware-tanzu/velero/pull/9357/files#r2450281419, let's move the `GetIncrementalSize()` function to this file, where we have both the `snapshotInfo` which takes the incremental size and the `progress` where we can get the factors to calculate the `incrementalSize`.